source/uwp/winui3/SimpleVisualizer/Assets/weather.json

```diff
@@ -0,0 +1,215 @@
+{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
```

Comment:
is there some way you can link this file from the samples folder rather than duplicating it?

Reply:
Done.